static/js/components/SourceTableFilterForm.jsx

```diff
@@ -209,15 +209,13 @@ const SourceTableFilterForm = ({ handleFilterSubmit }) => {
             Time Last Detected (UTC)
           </Typography>
           <TextField
-            disabled
```

Comment:
these are all disabled for now since they cause major slowness to the app. We're working on workarounds.